angular/core/components/group_page/group_theme/group_theme.haml

```diff
@@ -1,6 +1,6 @@
 .group-theme
-  .group-theme__cover{ng-style: "coverStyle()"}
-    .group-theme__upload-photo{ng-mouseover: "themeHoverIn()", ng-mouseleave: "themeHoverOut()", ng-if: "canUploadPhotos()"}
+  .group-theme__cover.group-theme__trigger-upload{ng-style: "coverStyle()", ng-mouseover: "themeHoverIn()", ng-mouseleave: "themeHoverOut()", ng-if: "canUploadPhotos()"}
```

Comment:
Also, I'm not that sure you need javascript for this. What if we did something like

```
.group-theme__cover           group-theme__upload-photo { display: none; }

.group-theme__cover:hover group-theme__upload-photo { display: block; }
```
?